keps/sig-node/4438-container-restart-termination/README.md

```diff
@@ -207,13 +207,19 @@ the restart during Pod termination.
 List the specific goals of the KEP. What is it trying to achieve? How will we
 know that this has succeeded?
 -->
+The following behaviors should be maintained during pod termination:
```

Comment:
is there some context that informed this decision (and the subsequent bump of version this KEP is going through?) Do you think that could be added here? something like:

```

In the initial implementation of this KEP, we attempted ... but it was found that ... and so we're choosing to ...
```

Reply:
@haircommander isn't it already described in the Motivation part right above?

Reply:
ah I see I misunderstood what was being updated

Reply:
broken English 👍